### ISSUE TITLE
Reconsider placement of accessibility statement in Preferences dialog.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/joist#838]
Now that we're moving sim-specific options to the Preferences dialog in #837 ....

Here's the _General_ tab for Geometric Optics:

<img width="779" alt="screenshot_1807" src="https://user-images.githubusercontent.com/3046552/186201964-9e60c0da-c9f9-4403-9b56-59659814f159.png">

In this context, the 2 paragraphs at the bottom seem out of place, and potentially confusing. These controls have nothing to do with accessibility, and the paragraphs would seem to be a description of those controls.

Discussing this with @jessegreenberg, these paragraphs were placed in the General tab when the Preferences dialog was accessibility-centric.  Now that we're using it for preferences that are outside the scope of accessibility, perhaps we should reconsider where these important paragraphs live.  One possibility would be on a new _Accesibility_ tab (similar to where such preferences are presented in macOS, Windows, and many applications.)

One other note... The last paragraph is:
> "To find other simulations with inclusive features, look for Access and Inclusion on the simulation filter page and filter by inclusive feature."

For someone who is using a downloaded version of the sim, this will be confusing. What is "the simulation filter page"?  This paragraph should probably say something about "on the PhET website".

**Comment by jessegreenberg** [comment]
> For someone who is using a downloaded version of the sim, this will be confusing

This makes sense to me!

> In this context, the 2 paragraphs at the bottom seem out of place, and potentially confusing.

Each tab can generally include features related to access and inclusion so these paragraphs describe the dialog as a whole. Now that we have more examples of sim-specific controls in the general tab, I can see how the mix of content may not be best. I don't think "General" is the best tab name for simulation specific controls.

I wanted to propose a layout like this where we have a landing tab that describes access and inclusion, and a "Simulation" tab for simulation controls.

https://phet-dev.colorado.edu/html/jg-tests/preferences/geometric-optics_en_phet.html

@terracoda @arouinfar @emily-phet @kathy-phet can you please review the recommendations in this issue and comment on what should be done next?

**Comment by terracoda** [comment]
I agree that there needs to be some changes to the language. There is room for improvement.

Before we start adjusting the words and the placement of the words, let's agree to not put some features or options in an "accessibility" category.

Some features may be required to make a sim accessible to a particular learner, but the same features are almost never exclusively for accessibility. 

While PhET sims have always been flexible in many ways, we are creating sims that are more flexible than ever! With options in the Preferences Menu our sims can bend and stretch to meet the needs of more learners and more learning contexts.

I would like a Preferences Menu that is clear and simple and does not have a proliferation of tabs. I am confident we can get there.

**Comment by terracoda** [comment]
@jessegreenberg, I think you are onto something, but there is room for improvement.

**Comment by kathy-phet** [comment]
> I wanted to propose a layout like this where we have a landing tab that describes access and inclusion, and a "Simulation" tab for simulation controls.

@jessegreenberg - I was also thinking this could work really well, but would propose the tab be called "Overview", not Accessibility. And the next tab be called Simulation or Sim-Specific

**Comment by pixelzoom** [comment]
In https://github.com/phetsims/joist/issues/838#issue-1348168341, I showed the sim-specific preferences for Geometric Optics.  In discussion with @kathy-phet, she proposed I might think of those preferences as being loosely related to inclusion, because they help tailor the sim for a specific age group.   I think that's a stretch, but I can imagine that for those specific controls. But I don't think that's true of sim-specific controls in general.
  
So here's another example, the sim-specific preferences from Molecule Polarity (run with `?realMolecules`).  The controls here choose between different conventions (standards) for representing dipole charge and isosurface.  In this case, I really can't find a way to relate the paragraphs to these controls. And I think it does a disservice to both the controls and the paragraphs (both of which are important) by combining them in this odd manner.

<img width="833" alt="screenshot_1823" src="https://user-images.githubusercontent.com/3046552/186570842-dc636ace-5c85-4e4a-a73d-ab789563b5c2.png">

**Comment by terracoda** [comment]
Ok, here's a sample re-write that may be general enough for an opening Tab, whatever it gets called

**Proposed Wording:**
P: The Preferences Menu provides access to all available presentation settings and features for a given simulation. Some features make the simulation more inclusive and can support accessibility for learners with diverse needs and within diverse environments. Explore this menu to review or change the default settings.

P: To find simulations with inclusive features, look for the Access and Inclusion filter on the PhET website.

**Comment by terracoda** [comment]
Since some simulations have several options, we may indeed indeed need an extra opening tab, and an aptly-named  sim-specific tab.

The place that gets fuzzy for me, is when sim-specific settings cross-over with the other tabs (i.e., the "Visual Tab"). But maybe this kind of cross-over can be worked out relatively easily by the sim designers.

**Comment by terracoda** [comment]
Brainstorming ideas for Sim Options or Sim Settings
Note that "tab" is spoken by a screen reader and included in the voiced tab name, though not visually displayed.
- Sim-Specific Options/Settings Tab
- Simulation Tab
- Simulation Options/Settings Tab
- Lesson Options/Settings Tab
- Teacher Options/Settings Tab
- Lesson Tab
- Teacher Tab
- Basic Sim Settings/Options Tab
- Sim Features Tab

I am not familiar with how the sim-specific settings are used. This is a brain storm.

**Comment by jessegreenberg** [comment]
There are a lot of classes throughout now with `GeneralPreferences` in the name. They should be renamed now.

**Comment by jessegreenberg** [comment]
OK, the "General" tab has been removed and replaced with "Overview" and "Simulation" tabs with some related code cleanup.

> The blurb needs some wordsmithing. Options to consider

@arouinfar back to you for this part or to reassign to anyone else.

@zepumph can you please spot check? Most things were straight forward, but I used `Simulation` in class/option/type names. Was slightly worried about confusion because "Simulation" is overloaded for us. But thought it was best to match other preferences naming since that is the name of the tab.

**Comment by zepumph** [comment]
You are so very talented. This is amazing. I liked above issues that I think are high priority that I found while reviewing this, but I have no recommendations here. Let me know if you need anything else.

**Comment by arouinfar** [comment]
This issue remains open because there has been some discussion about improving the wording of the "Overview" blurb, but we did not reach a conclusion. Some options appear in https://github.com/phetsims/joist/issues/838#issuecomment-1230732910.

When work resumes on the Preferences Dialog (https://github.com/orgs/phetsims/projects/61), the team should decide how to proceed. For now, I'm unassigning myself.

**Comment by terracoda** [comment]
We no longer have a named "Access and Inclusion" section on the Simulation filter page, so I think the content changes are higher priority now.

The current Overview Tab has this content:

> We are adding features to our simulations to make them more inclusive. Some of these features support accessibility for learners with diverse needs and within diverse environments. Explore this menu to review or change the default presentation settings.
>
>To find other simulations with inclusive features, look for Access and Inclusion on the simulation filter page and filter by inclusive feature.

<img width="766" height="179" alt="Image" src="https://github.com/user-attachments/assets/88c8c625-84f1-4af2-aebf-5bd31f8d9ae0" />

**I think the follow change could work well:**
We are adding features to our simulations to make them more inclusive. Some of these features support accessibility for learners with diverse needs. Explore this menu to review or change the default presentation settings.

Look for Inclusive Features on the Simulations Filter page of the PhET website to find all simulations with inclusive features.

**Comment by terracoda** [comment]
Oops, just reviewing https://github.com/phetsims/joist/issues/838#issuecomment-1227375450.
Given that, here's what I suggest:

**Proposed New Wording:**
P: The Preferences Menu provides access to all available presentation settings and features for a given simulation. Some features make the simulation more inclusive and can support accessibility for learners with diverse needs. Explore this menu to review or change the default settings.

P: Look for Inclusive Features on the Simulations Filter page of the PhET website to find all simulations with inclusive features.

**Comment by arouinfar** [comment]
@brent-phet this issue overlaps https://github.com/phetsims/joist/issues/1023, and is possibly a duplicate.

**Comment by brent-phet** [comment]
Duplicate with #1023. Closing.

**Comment by kathy-phet** [comment]
@brent-phet - Before closing, can you move the TO DO we added here to the new issue, since we are to check with @jbphet and such?
